olive/data/template.py

```diff
@@ -47,6 +47,15 @@ def huggingface_data_config_template(model_name, task, **kwargs) -> DataConfig:
         and other arguments in
             - olive.data.component.load_dataset.huggingface_dataset
             - olive.data.component.pre_process_data.huggingface_pre_process
+                - `remap_labels`: true | false, whether to remap the labels with huggingface model config
+                - `model_config_path`: str, path to the model config file
+                - others is used for huggingface tokenizer
+                e.g.:
+                "component_kwargs": {
+                    "pre_process_data": {
+                        "remap_labels": true
```

Comment:
>remap_labels [](http://example.com/codeflow?start=25&length=12)

could we make this more general so that we can support cast_column/remove_column/select_columns/map/filter/shuffle/shard etc in future?

The current name seems like a case-by-case fix

Reply:
Thanks, add it into TODO items to unblock Xiaoyu's work. 
If we meet similar problem later, we can circle back for better solution.